packages/console/src/widget.ts

```diff
@@ -474,6 +483,43 @@ class CodeConsole extends Widget {
     }
   }
 
+  /**
+   * Handle the `'focusin'` event for the widget.
+   */
+  private _evtFocusIn(event: FocusEvent): void {
+    // Add a class to designate that either the console element or
+    // the cell editor has focus.
+    this.addClass('jp-mod-focus');
+
+    // Check whether the console itself is getting focused.
+    if (event.target === this.node) {
+      this._content.addClass('jp-mod-focus');
```

Comment:
Do we need separate `jp-mod-focus` classes on the content and parent divs? I am a bit concerned about us starting to have our own focus logic in various parts of the app. Need to make sure we only this abstraction when it is needed and regular focus selectors otherwise.